src/hypothesis/extra/django/models.py

```diff
@@ -115,7 +115,7 @@ def __init__(self, model, mappings):
             strategy=st.fixed_dictionaries(mappings))
 
     def __repr__(self):
-        return u'ModelStrategy(%s)' % (self.model.__name__,)
+        return 'ModelStrategy(%s)' % (self.model.__name__,)
```

Comment:
Hm -- what do you mean? `__repr_` can return whatever it wants, but its return type on Py2 is bytes (and Py3 unicode, so it's a native string thing). It's not the greatest idea to return non-ASCII things in Py2 because who wants to deal with possible terminal issues when they want to just debug some stuff, but it "works" / "is allowed" if someone decides to. Or have I forgotten things?

It's less important I think to actually support non-ASCII things in these places ("nice" as that might be) as it is to make sure not to start *mixing in* implicit conversion for someone who might encounter the return values and then start seeing unicode errors even though they're doing things "correctly" -- but you're definitely right that this is tiny-percentage nonsense, I just sort of covered a few of 'em at once.